scripts/quests/otherAreas/Better_The_Demon_You_Know.lua

```diff
@@ -0,0 +1,155 @@
+-----------------------------------
+-- Better The Demon You Know
+-----------------------------------
+-- Log ID: 4, Quest ID: 105
+-- Koblakiq !pos -64.851 21.834 -117.521
+-- qm2      !pos 19.4 -24.141 19.185
+-----------------------------------
+local zvahlID = zones[xi.zone.CASTLE_ZVAHL_BAILEYS]
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.BETTER_THE_DEMON_YOU_KNOW)
+
+quest.reward =
+{
+    item = xi.item.GOBLIN_GRENADE,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+                player:hasCompletedQuest(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.FOR_THE_BIRDS) and
+                not xi.quest.getMustZone(player, xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.FOR_THE_BIRDS)
+        end,
+
+        [xi.zone.OLDTON_MOVALPOLOS] =
+        {
+            ['Koblakiq'] = quest:progressCutscene(20, 0, xi.item.DEMON_PEN),
+
+            onEventFinish =
+            {
+                [20] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_ACCEPTED
+        end,
+
+        [xi.zone.OLDTON_MOVALPOLOS] =
+        {
+            ['Koblakiq'] =
+            {
+                onTrigger = function(player, npc)
+                    local progress = quest:getVar(player, 'Prog')
+                    local waitTime = quest:getVar(player, 'Wait')
+
+                    if progress == 0 then
+                        return quest:event(21, 0, xi.item.DEMON_PEN):setPriority(101) -- Reminder to get Demon Pen
+                    elseif progress == 1 then
+                            if GetSystemTime() >= waitTime + 60 then
+                                return quest:progressCutscene(24) -- Go to Castle Zvahl Baileys
+                            else
+                                return quest:event(23):setPriority(101) -- Wait Longer
+                            end
+                    elseif
```

Comment:
this specific elseif is dead now (I assume it's the one above it after clay's comment)